ospf6d/ospf6_asbr.c

```diff
@@ -1505,6 +1540,145 @@ static void ospf6_redistribute_show_config(struct vty *vty, struct ospf6 *ospf6,
 		vty_out(vty, "Total %d routes\n", total);
 }
 
+static void ospf6_redistribute_default_set(struct ospf6 *ospf6, int originate)
+{
+	struct prefix_ipv6 p;
+	struct in6_addr nexthop;
+	int cur_originate = ospf6->default_originate;
+
+	p.family = AF_INET6;
+	p.prefixlen = 0;
+
+	memset(&p.prefix, 0, sizeof(p.prefix));
```

Comment:
These `memset` could be replaced with the faster in declaration zeroing:

```c
struct prefix_ipv6 p = {};
struct in6_addr nexthop = {};
```

Reply:
Fixed